software/shkeeper

```diff
@@ -0,0 +1,11 @@
+name: SHKeeper.io
+website_url: https://shkeeper.io/
+source_code_url: https://github.com/vsys-host/shkeeper.io
+description: A self-hosted and open-source cryptocurrency payment processor. The unique combination of gateway and merchant. It allows you to accept payments in multiple cryptocurrencies without fees and intermediaries.
```

Comment:
```suggestion
description: A self-hosted and open-source cryptocurrency payment processor. An unique combination of gateway and merchant, it allows you to accept payments in multiple cryptocurrencies without fees and intermediaries.
```

Reply:
No issues with this change

Reply:
This may be better
```suggestion
description:  Cryptocurrency payment processor with the unique combination of gateway and merchant allowing you to accept payments in multiple cryptocurrencies without fees and intermediaries."
```